src/js/media-controller.js

```diff
@@ -1137,14 +1137,17 @@ const getShowingCaptionTracks = (controller) => {
 
 const getStreamType = (controller) => {
   const { media } = controller;
-  
-  if (!media) return undefined;
-  
+
+  if (!media) return StreamTypes.UNKNOWN;
```

Comment:
if there is no media could the stream type algo have run? should this stay undefined?

will this cause a new attribute to be added `media-stream-type=unknown`?

Reply:
> if there is no media could the stream type algo have run? 
Yes
> should this stay undefined?
Maybe (See below)
> will this cause a new attribute to be added `media-stream-type=unknown`?
Yes.

The current implementation strictly conforms with expected values from https://github.com/video-dev/media-ui-extensions/issues/3. I do think for this case there is a compelling argument to instead deviate (@heff called this out in the proposal discussion), but we want to be consistent here one way or another, either:

1. **_always_** translate `"unknown"` cases to `undefined`
2.. _**always**_ treat `undefined` and similar cases as `"unknown"`

Eventually, we may want to distinguish between these two scenarios in Media Chrome, but for now, this will only add complexity for UI devs as far as I can tell. I'll refactor to always use `undefined` (aka (1), above) per your / @heff's callout just to reduce the attributes footprint, though, in general, I think it's good to err on the side of predictable consistency with media-ui-extensions.